llvm/test/CodeGen/AMDGPU/llc-pipeline.ll

```diff
@@ -60,8 +60,6 @@
 ; GCN-O0-NEXT:        Cycle Info Analysis
 ; GCN-O0-NEXT:        Uniformity Analysis
 ; GCN-O0-NEXT:        Unify divergent function exit nodes
-; GCN-O0-NEXT:        Lazy Value Information Analysis
-; GCN-O0-NEXT:        Lower SwitchInst's to branches
```

Comment:
Removing this is nice. I prototyped an alternative way to do it: https://github.com/jayfoad/llvm-project/commit/c03cbd4d2f584d24e2100fee529f2ba1d18c345d

See also #59830